pkg/jaeger/query/trace_query.go

```diff
@@ -31,20 +31,20 @@ const (
 		s.trace_state 			trace_states,
 		s_url.url 				schema_urls,
 		o.span_name     		span_names,
-	   	ps_trace.jsonb(s.resource_tags) 	resource_tags,
-	   	ps_trace.jsonb(s.span_tags) 		span_tags,
+		_ps_trace.tag_map_denormalize(s.resource_tags) 	resource_tags,
+		_ps_trace.tag_map_denormalize(s.span_tags) 		span_tags,
 	   	array_agg(e.name ORDER BY e.event_nbr) FILTER(WHERE e IS NOT NULL)			event_names,
 		array_agg(e.time ORDER BY e.event_nbr) FILTER(WHERE e IS NOT NULL)			event_times,
 	   	array_agg(e.dropped_tags_count ORDER BY e.event_nbr) FILTER(WHERE e IS NOT NULL)	event_dropped_tags_count,
-	   	jsonb_agg(ps_trace.jsonb(e.tags) ORDER BY e.event_nbr) FILTER(WHERE e IS NOT NULL)	event_tags,
+		array_agg(_ps_trace.tag_map_denormalize(e.tags) ORDER BY e.event_nbr) FILTER(WHERE e IS NOT NULL)	event_tags,
```

Comment:
I don't understand this. The old thing returns a `jsonb` array:
```JSON
[{"a":1}, {"a":2}, ...]
```
The new thing is actually array of `jsonb`s, which is not really the same thing (former is `jsonb` latter is `jsonb[]`)

Reply:
ps_trace.jsonb is pretty much _ps_trace.tag_map_denormalize

Reply:
Sorry, I didn't realize that we actually change the type as well. This change of type is intended.